ecs/sdk.go

```diff
@@ -364,7 +368,23 @@ type stackResource struct {
 	Status    string
 }
 
-func (s sdk) ListStackResources(ctx context.Context, name string) ([]stackResource, error) {
+type stackResourceFn func(r stackResource) error
+
+type stackResources []stackResource
+
+func (resources stackResources) apply(awsType string, fn stackResourceFn) error {
+	for _, r := range resources {
+		if r.Type == awsType {
+			err := fn(r)
+			if err != nil {
```

Comment:
we can use multierr here, see for example https://github.com/gtardif/compose-cli/blob/main/cli/cmd/rm.go#L60

Reply:
Initial usage expect to only match a single resource, but makes sense